src/network.js

```diff
@@ -578,18 +596,17 @@ Network.getWorkerSharedFunctions = function() {
   //  using the .toString() method
 
   // Load and name the train function
-  var train_f = Trainer.prototype.train.toString();
-  train_f = train_f.replace('function (set', 'function train(set') + '\n';
+  var train_f = 'train: ' + Trainer.prototype.train.toString() + ',\n';
```

Comment:
are those (this and next 2) changes actually required to bring a support of node.js trainAsync?